### ISSUE TITLE
[FR]: see all active message filters for a feed?

### ISSUE THREAD

**Issue by Owyn** [martinrotter/rssguard#1473]
### Brief description of the feature request

![image](https://github.com/user-attachments/assets/be77468c-7ec9-4924-9aaa-9d039f92496f)

Now there's a text which displays how many filters work for a specific feed but... which ones?!?!

Now my feed for some unknown reason doesn't update but if I add its url again - it gets all the new stuff, so it must be filters then but... I have like 30 and it's like hell finding out which ones are enabled for this feed

so, please make a button to show all active filters for a selected feed,
if it already shows that there are 4 filters for it - it should be easy to just open the filters dialogue while hiding all non-active (for this feed) filters, right?

**Comment by martinrotter** [comment]
this is already there

![image](https://github.com/user-attachments/assets/58a64485-3199-421e-9826-334f6ec14644)

**Comment by Owyn** [comment]
> this is already there
> 


Do you mean this? 
![image](https://github.com/user-attachments/assets/1461ebae-12c6-492d-a12f-989886e65690)

Even on your screen it's not helpful cuz the names are the same lol,
I mean - show an actual list of filters you could click to turn off or edit

**Comment by martinrotter** [comment]
yes names are same because i added two filters with the same name just for the screenshot

**Comment by Owyn** [comment]
It'd be ideal if these were clickable and it took us directly to the filter editing with that specific filter we clicked opened already:

![Image](https://github.com/user-attachments/assets/7d361582-f15b-49d1-a593-97328bb3d751)

not sure if it's possible with the legacy article viewer tho

or inside the right click menu just add "view filters for this feed" so clicking it would open the filters menu with only filters active for this feed shown and the rest - hidden

![Image](https://github.com/user-attachments/assets/69867f84-67ee-4037-bc48-ced12e1406c3)

Or if not here then in the feed edit window - a list of filters to open right away would be super-helpful: 

![Image](https://github.com/user-attachments/assets/5fadbd1c-d0e2-4133-8998-1d4a3eb85b74)
e.g. as a new tab (instead of experimental which could go inside "miscellaneous")
